src/pages/Search/Search.js

```diff
@@ -1,23 +1,61 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import config from '../../config';
 import Page from '../../components/Page';
-import HeatMap from '../../components/HeatMap';
-import ResultsTable from '../../components/ResultsTable';
 import InputForm from '../../components/InputForm';
 import Headline from '../../components/Headline';
-
+import Spinner from '../../components/Spinner';
 
 export default function Search() {
+  const { slug } = useParams();
+
+  const [data, setData] = useState([]);
+  const [loading, toggleLoading] = useState(false);
```

Comment:
The name should better be `setLoading`. If it is called `toggleLoading` I would expect it to be called without parameter like `toggleLoading()` and the toggling logic would be handled by the function itself